packages/nuxt/package.json

```diff
@@ -39,7 +39,9 @@
     "access": "public"
   },
   "peerDependencies": {
-    "nuxt": "3.x"
+    "nuxt": ">=3.7.0 || 4.x",
+    "nitropack": ">=2.10.0",
+    "ofetch": ">=1.4.0"
```

Comment:
Hmm, can we do this? Does this not enforce users to specify this version in their own `package.json`, which they probably do not have? 🤔

Reply:
The user does not have to explicitly install it, it just makes sure the dependency tree has the correct version. But if there is a conflicting peer, users might have to install it before npm v6: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependencies

Reply:
Hmm...probably it's better to just not include those peer deps, just to be sure. I'll add them to the docs and list them as a requirement.